symengine/rational.h

```diff
@@ -156,15 +160,23 @@ class Rational : public Number
     inline RCP<const Number> divrat(const Integer &other) const
     {
         if (other.i == 0) {
-            throw DivisionByZeroError("Division By Zero");
+            if (this->i == 0) {
+                throw NotImplementedError("0/0 is NaN. Yet to be implemented");
+            } else {
+                return ComplexInf;
+            }
         } else {
             return from_mpq(this->i / other.i);
         }
     }
     inline RCP<const Number> rdivrat(const Integer &other) const
     {
         if (this->i == 0) {
-            throw DivisionByZeroError("Division By Zero");
+            if (other.is_zero()) {
```

Comment:
@isuruf  pardon .. i meant rational.h ..
like this should be changed to other.i as well ?